app/BotMan/Conversation/SpotifyUrlConversation.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\BotMan\Conversation;
+
+use BotMan\BotMan\Messages\Conversations\Conversation;
+use BotMan\BotMan\BotMan;
+use App\SpotifyTrackConverter;
+use App\Spotify\Track;
```

Comment:
Nitpick: I like to alphabetize my imports. It's super easy with PHPStorm, and I'm sure there's a key combo (maybe via a plugin) for VIM.

Reply:
I started doing this after reading this review.  I like it a lot, actually.